cwf/gateway/integ_tests/gy_enforcement_test.go

```diff
@@ -35,7 +35,7 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
-func ocsCreditExhaustionTestSetup(t *testing.T) (*TestRunner, *RuleManager, *cwfprotos.UEConfig) {
+func ocsAndPcrfCreditExhaustionTestSetup(t *testing.T) (*TestRunner, *RuleManager, *cwfprotos.UEConfig) {
```

Comment:
I changed the name because it was hard for me to find where the PCRF was actually configured